rbac/processor/role/role_admins.py

```diff
@@ -166,3 +168,18 @@ def apply_reject_remove(header, payload, state):
             header, payload, state
         )
     )
+def hierachical_approve(header, payload, state):
```

Comment:
Consistent but incorrect spelling across this commit for "hierarchical". Please address.

Reply:
haha. english is not my first language. will fix. thanks for pointing out